.travis.yml

```diff
@@ -18,7 +18,7 @@ cache:
 before_script:
   - phpenv config-add .travis.php.ini
   - composer self-update
-  - composer install --prefer-dist
+  - COMPOSER_MEMORY_LIMIT=-1 composer install --prefer-dist
```

Comment:
I think it would be better to have a more sane default. This solution would prevent us from getting a descent response why composer did fail. I don't mind if it's getting fixed later.

Reply:
Thanks for the feedback! Opened a TD ticket on the stepup backlog to look into this. For now to keep momentum, merging this as is.

https://www.pivotaltracker.com/story/show/169599237